src/ui/css/vars.scss

```diff
@@ -36,8 +36,8 @@ $header-accent-color: $link-color;
 $header-text-not-active-color: transparentize($white, 0.5);
 
 // Category Colors
-$category-colors: #fa5252, #e64980, #be4bdb, #7950f2, #4c6ef5, #228ae6, #15aabf,
-  #12b886, #40c057, #82c91e, #fab005, #fd7e14;
+$category-colors: $teal-70, $red-50, $magenta-60, $blue-50, $orange-70,
```

Comment:
Using the shiny new accessibility tools, I see that the following color / text combos fail in their contrast ratios:

* teal-70
* red-50
* magenta-60
* blue-50
* green-60
* teal-60
* green-70
* orange-50

Interestingly, the Photon design page claims that some combos listed above should not fail the contrast ratio. This might be due to the smaller font size used in our category layout.

Here's an example of how I was using the accessibility tool:

<img width="382" alt="Screenshot 2019-10-23 10 05 41" src="https://user-images.githubusercontent.com/55398/67407572-535f9180-f57d-11e9-9495-9422f9508bb0.png">
<img width="341" alt="Screenshot 2019-10-23 10 05 59" src="https://user-images.githubusercontent.com/55398/67407578-55295500-f57d-11e9-821c-ba6286d80750.png">

Reply:
Thanks. I tried to use colors that I could actually... see (I also have kind of colorblindness). Would you have suggestions for other colors or should I just go ahead and pick some that have a better contrast?

Reply:
According to [the page referenced in the devtools](Active user interface components and graphical objects such as icons and graphs), `3:1` should be enough for "Active user interface components and graphical objects such as icons and graphs". Does a button qualify as an "active UI component"?

If so, then only one color seems too low (green-50).